templates/places/opening.html

```diff
@@ -0,0 +1,14 @@
+<!-- {% extends 'base.html' %}
```

Comment:
See my comment above about this feature.

Reply:
This has been taken care of.